http_prompt/printer.py

```diff
@@ -0,0 +1,20 @@
+import click
+
+
+class Printer():
```

Comment:
@fogine Instead of using `sys.stdout` directly, I think you should use `click.echo_via_pager` whenever you want to write to stdout. My bad in the example code.